fastf1/api.py

```diff
@@ -97,17 +97,28 @@ class Cache:
           running your scripts,  as parsing of the data is computationally
           expensive. Stage 2 caching is only used for some api functions.
 
-    Most commonly, you will enable caching right at the beginning of your script:
+    You can explicitly configure right at the beginning of your script:
 
         >>> import fastf1
         >>> fastf1.Cache.enable_cache('path/to/cache')  # doctest: +SKIP
         # change cache directory to an exisitng empty directory on your machine
         >>> session = fastf1.get_session(2021, 5, 'Q')
         >>> # ...
 
-    Note that you should always enable caching except for very rare
-    circumstances which are usually limited to doing core developement
-    on FastF1.
+    An alternative way to set the cache directory is to configure an
+    environment variable `FASTF1_CACHE`. However, this value will be
+    ignored if `Cache.enable_cache()` is called.
+
+    If no explicit location is provied, Fast-F1 will use a default location
+    depending on operating system.
+
+        - Windows: `%LOCALAPPDATA%\\Temp\\fastf1`
```

Comment:
Sorry, there is still a minor problem that I just spotted when checking the doc build. Annoyingly enough, you need to double escape backslashes here. Therefore, you need FOUR backslashes in the docstring to render a single bachslash in html.

Reply:
Good catch! I certainly wasn't expecting `\\\\` :sweat_smile: 

Fixed!